.prettierrc

```diff
@@ -1,5 +1,9 @@
 {
-  "semi": false,
-  "singleQuote": true,
-  "trailingComma": "es5"
+	"semi": true,
+	"singleQuote": true,
+	"trailingComma": "es5",
+	"bracketSpacing": true,
+	"printWidth": 120,
+	"useTabs": true,
+	"tabWidth": 4
```

Comment:
Agree with @MylesBorins , I’d prefer to use 2 spaces instead here.

IMHO, we’d better be consistent with .editorconfig:

https://github.com/nodejs/nodejs.dev/blob/0194c772257d567d441f2b727d1f4b0b9b5d447b/.editorconfig#L1-L9

Reply:
Done!